manager/manager_test.go

```diff
@@ -400,3 +400,110 @@ func TestManagerLockUnlock(t *testing.T) {
 	// error.
 	<-done
 }
+
+// If the root CA materia is updated in the memory store, a manager will update its own
```

Comment:
`material`

(I don't get any pleasure out of pointing out typos, but if I don't do it, someone else will open a PR, and I prefer to preempt those.)